BlueEastCode/bluerain-app-settings
# Feature/improvements

## What I did
- Fixed bugs related to Layouts
- Added Main Header to the layout

## How to test

Is this testable with jest or storyshots?
Yes, however right now its only tested manually in storybooks.

Are any of the existing tests are failing ?
Needs to be tested.

Does this need an update to the documentation?
Yes, but its not done at the moment.

Does it have breaking changes?
No.

If your answer is yes to any of these, please make sure to include it in your PR.

@bjavaid @amnajunaid please review the changes and merge them.
